docs/frontend.md

```diff
@@ -360,13 +302,13 @@ The [shopping-list-redux-hoc](https://github.com/reimagined/resolve/tree/master/
 
 The **@resolve-js/react-hooks** library provides React hooks that you can use to connect React components to a reSolve backend. The following hooks are provided.
```

Comment:
1. `The **@resolve-js/react-hooks** library provides React hooks that you can use to connect React components to a reSolve backend.`

We should update this sentence so that it doesn't use "provides". Can we change it to "includes"?

2. `The following hooks are provided.`

Please change the period at the end to a colon.

Reply:
Changed to "provides" to "includes" . 

>Please change the period at the end to a colon.

Done